### DISCUSSION TITLE
Portable version

### DISCUSSION THREAD

**Discussion by VNRARA** [File-New-Project/EarTrumpet#733] [Q&A] [answered]
Would you happen to have a portable version?

I want to use it on my Workpc and because of policy the store version cannot be installed.

**Comment by riverar** [comment, marked as answer]
We have a Chocolatey package you can download and unzip manually if needed https://chocolatey.org/packages/eartrumpet/

**Reply by starsoccer** [reply]
Are settings and options though all stored in the same location as the zip, or in app data?
